docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,95 @@ weight = 4
+
+__TV mode:__ The side mmenu is hidden/removed. The navbar, row and panel controls appear at the top of the screen.
 
-Playlists can also be manually controlled utilizing the Playlist controls at the top of screen when in Playlist mode.
+A playlist enters __TV mode__ automatically after one minute of user inactivity. To toggle it manually, 
+use the `d v` sequence shortcut, or append the parameter `?inactive` to the dashboard URL. Restore the navbar and controls with any mouse movement or keyboard action.
 
-Click the stop button to stop the Playlist, and exit to the current Dashboard.
-Click the next button to advance to the next Dashboard in the Playlist.
-Click the back button to rewind to the previous Dashboard in the Playlist.
+__Kiosk mode:__ The side menu, navbar, row and panel controls are completely hidden/removed from view. 
+
+To put a playlist into __Kiosk__ mode, use the `d k` shortcut after the playlist has started. Toggle the playlist out of kiosk mode with the same shortcut.
+
+## Control a playlist
 
-## TV or kiosk mode
+To control a playlist:
 
-In TV mode the top navbar, row and panel controls will all fade to transparent.
+1. While in __Normal__ or __TV__ mode, locate the Playlist navbar and controls at the top of the screen.
 
-This happens automatically after one minute of user inactivity but can also be toggled manually
-with the `d v` sequence shortcut, or by appending the parameter `?inactive` to the dashboard URL. Any mouse movement or keyboard action will
-restore navbar and controls.
+   Note: A playlist cannot be controlled manually in __Kiosk__ mode.
 
-Another feature is the kiosk mode - in kiosk mode the navbar is completely hidden/removed from view. This can be enabled with the `d k`
-shortcut.
+2. Click the __next__ button (right arrow) to advance to the next dashboard. 
+3. Click the __back__ button (left arrow) in the navbar to return to the previous dashboard. 
+4. Click the square __Stop__ button to stop the playlist. This will exit to the current Dashboard.
 
-To put a playlist into kiosk mode, use the `d k` shortcut after the playlist has started. The same shortcut will toggle the playlist out of kiosk mode.
+>Shortcut: Click the __Esc__ key to stop the playlist from your keyboard.
 
-### Link to a playlist in kiosk mode
+### Link to a playlist in Kiosk mode
 
 To create a link to a playlist with kiosk mode enabled:
 
-1. Copy the Start URL (right click  the **Play** button and choose **Copy link address**).
+1. Copy the Start URL (right click the __Play__ button and choose __Copy link address__).
 2. Add the `?kiosk` parameter to the URL.
 
-For example, to open the first playlist on the Grafana Play site in kiosk mode, the URL should look like this:
+For example, to open the first playlist on the Grafana Play site in Kiosk mode, the URL should look like this:
 [http://play.grafana.org/playlists/play/1?kiosk](http://play.grafana.org/playlists/play/1?kiosk).
```

Comment:
I feel like this is probably referring to an older version. I assume that you would click **Start playlist**, and right click one of the modes in the drop down:

<img width="1073" alt="Screen Shot 2019-10-03 at 11 43 55" src="https://user-images.githubusercontent.com/8396880/66116629-547f4d80-e5d3-11e9-8401-6b9fb4a128db.png">